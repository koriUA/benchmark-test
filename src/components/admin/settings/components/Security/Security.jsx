import React from 'react';

const Security = () => <div>Here goes content for Security</div>;

export default Security;
