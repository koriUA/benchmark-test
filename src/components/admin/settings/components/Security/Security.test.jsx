import React from 'react';
import { mount } from 'enzyme';

import Security from './Security';

describe('Security', () => {
  test('should render the Security', () => {
    const component = mount(<Security />);
    expect(component).toMatchSnapshot();
  });
});
