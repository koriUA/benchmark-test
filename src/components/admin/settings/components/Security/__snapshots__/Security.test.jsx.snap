// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Security should render the Security 1`] = `
<Security>
  <div>
    Here goes content for Security
  </div>
</Security>
`;
